packages/cdk8s-plus/src/duration.ts

```diff
@@ -1,12 +1,9 @@
-import { Token } from 'constructs';
```

Comment:
For simplicity, I just dropped the entire token handling in the `Duration` class. Do we really need it?

Reply:
Okay